src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-multisite-sync-flow-modal/rgw-multisite-sync-flow-modal.component.ts

```diff
@@ -162,16 +153,29 @@ export class RgwMultisiteSyncFlowModalComponent implements OnInit {
       this.currentFormGroupContext.setErrors({ cdSubmitButton: true });
       return;
     }
+
+    const zones: ZoneType = { added: [], removed: [] };
+    if (this.groupType == FlowType.symmetrical) {
+      if (this.editing) {
+        zones.removed = this.flowSelectedRow.zones.filter(
+          (zone: string) => !this.zones.data.selected.includes(zone)
+        );
+        zones.added = this.zones.data.selected.filter(
+          (zone: string) => !this.flowSelectedRow.zones.includes(zone)
+        );
+      }
+      zones.added = zones.added.length > 0 ? zones.added : this.zones.data.selected;
```

Comment:
```suggestion
      zones.added = zones.added.length || this.zones.data.selected;
```

Reply:
I guess, this is not what I want, as this will give the array length instead of array data